### PR TITLE
Added German translations and admin config interface

### LINKED ISSUE #1: Configure defaults in Admin

### PR DESCRIPTION
**Pull request by pikim**

### CODE REVIEW

File: blueprints.yaml [hunk 1]
```diff
@@ -20,12 +20,30 @@ form:
   validation: loose
   fields:
     enabled:
-      type: toggle
       label: PLUGIN_ADMIN.PLUGIN_STATUS
+      type: toggle
```
Comment (Karmalakas) [inline blueprints.yaml]: Why the order change?

File: blueprints.yaml [hunk 2]
```diff
@@ -20,12 +20,30 @@ form:
...
       highlight: 1
       default: 0
       options:
-        1: PLUGIN_ADMIN.ENABLED
         0: PLUGIN_ADMIN.DISABLED
+        1: PLUGIN_ADMIN.ENABLED
```
Comment (Karmalakas) [inline blueprints.yaml]: Same question - why the change? I'd prefer seeing the default as the first option

File: blueprints.yaml [hunk 3]
```diff
@@ -20,12 +20,30 @@ form:
...
       validate:
         type: bool
+
+    processor:
+      label: Processor
```
Comment (Karmalakas) [inline blueprints.yaml]: Both labels (processor and formats) would need to have proper translation strings.
 Something like:
```
PLUGIN_TRANSLATE_DATE:
  fields:
    processor:
      label: Processor
    formats:
      label: Formats
      placeholder:
        key: Language
        value: Format
  format:
    F: ...
    M: ...
    l: ...
    D: ...
```

I guess it would require adjusting [the filter class](https://github.com/Karmalakas/grav-plugin-translate-date/blob/main/classes/Twig/FilterTd.php#L198) too.

Also would require adding info about at least formats field to the README

**Review by Karmalakas — CHANGES_REQUESTED** [review]

**Comment by Karmalakas** [comment]
Closes #1

File: blueprints.yaml (hunk 1 above)
Comment (pikim) [reply blueprints.yaml]: Because I like to have the label first. But it's not really important for me.

File: blueprints.yaml (hunk 2 above)
Comment (pikim) [reply blueprints.yaml]: Because I like to have it in increasing order. But it's not really important for me. By the way: according to the file the default is 0.

File: blueprints.yaml (hunk 3 above)
Comment (pikim) [reply blueprints.yaml]: I know. I just saw your notice in the readme and the issue tracker and made a minimal commit.

I think it will work out of the box as the config is stored in the exact same format as before.

File: blueprints.yaml (hunk 2 above)
Comment (Karmalakas) [reply blueprints.yaml]: I'm not even sure why I had default as disabled 😕 It should've been enabled from the beginning

File: languages.yaml [hunk 4]
```diff
@@ -1,10 +1,29 @@
+de:
+  PLUGIN_TRANSLATE_DATE:
+    F: ['Januar', 'Februar', 'März', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Dezember']
+    M: ['Jan', 'Feb', 'Mär', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dez']
+    l: ['Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag', 'Sonntag']
+    D: ['Mon', 'Die', 'Mit', 'Don', 'Fre', 'Sam', 'Son']
+
+    PROCESSOR: Prozessor
```
Comment (Karmalakas) [inline languages.yaml]: I would prefer to not mix up formats translations with other non-format strings. Also nested structure is more readable and future proof

**Review by Karmalakas — CHANGES_REQUESTED** [review]

File: languages.yaml (hunk 4 above)
Comment (pikim) [reply languages.yaml]: The convention is to have the plugin prefix followed by the translation element, e.g. PLUGIN_TRANSLATE_DATE.PROCESSOR. So if you don't like it, it would make sense to encapsulate the format translations imo.

Edit:
That would lead to:
```
en:
  PLUGIN_TRANSLATE_DATE:
    FORMATS:
      F: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
      M: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
      l: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
      D: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
      
    PROCESSOR: Processor
    DEFAULT_FORMATS: Default formats
    DEFAULT_FORMATS_HELP: Default format for date and time in the according language.
    PLACEHOLDER_KEY: Language
    PLACEHOLDER_VALUE: Format
```
and some changes in the code.

Comment (pikim) [reply languages.yaml]: Did you think about it? What would be your favorite way?

Comment (Karmalakas) [reply languages.yaml]: It's on my todo list. I would still prefer the way I explained earlier to make it easier for myself to maintain. I don't want to merge it the way it is currently and I didn't find the time to get to it yet 🙁

Comment (pikim) [reply languages.yaml]: What about going back one step and leaving the admin interface untranslated? That would also be okay for me and many other plugins also come only with an English admin interface.

Comment (Karmalakas) [reply languages.yaml]: I guess it would work for now. I might get to those some day though

**Comment by Karmalakas** [comment]
Closes #1

**Review by Karmalakas — APPROVED** [review]

**Comment by Karmalakas** [comment]
Thanks for the contribution. I've published a release and it should appear on GPM soon enough
